### PR TITLE
i18n(es): Translate `from-create-react-app.mdx`

### PR DESCRIPTION
**Pull request by Waxer59**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Translate `from-create-react-app.mdx`

### CODE REVIEW

File: src/content/docs/es/guides/migrate-to-astro/from-create-react-app.mdx [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+yarn astro add react
+yarn astro add react tailwind mdx
+# Usando PNPM
+pnpm astro add react
+yarn astro add react tailwind mdx
+```
+
+Si tu CRA requiere cualquier dependencia (p. ej. paquetes de NPM), debes instalarlos individualmente utilizando la línea de comandos o añadiéndolos manualmente al archivo `package.json` de tu nuevo proyecto de Astro, y luego ejecutando el comando de instalación. Ten en cuenta que muchas, pero no todas, las dependencias de React funcionarán en Astro.
+
+### Agrega los archivos de tu app existente
```
Comment (dreyfus92) [inline src/content/docs/es/guides/migrate-to-astro/from-create-react-app.mdx:96]:
```suggestion
### Agrega los archivos de tu aplicación existente
```

**Review by dreyfus92 — CHANGES_REQUESTED** [review]
Buen trabajo @Waxer59, te dejo unas pequeñas sugerencias.

**Comment by Waxer59** [comment]
Listo!, gracias por la revisión 🤠

**Review by dreyfus92 — APPROVED** [review]
LGTM ✅

**Review by Yan-Thomas — APPROVED** [review]
LGTM peeps, thanks for the great work 💜

**Pull request merged**